nf1_cellpainting_env.yml

```diff
@@ -15,6 +15,8 @@ dependencies:
 # this package must be hardcoded to work with Mac 
 - conda-forge::mysqlclient=1.4.6
 - conda-forge::openjdk
+# must specifiy scikit-image to avoid error in MeasureGranularity
```

Comment:
thank you for this documentation

Reply:
Yup! This will probably be fixed once the issue is reviewed from CellProfiler (or I make a PR to CellProfiler depending on bandwidth).